WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -333,6 +359,21 @@ final class EditableOrderViewModel: ObservableObject {
         stores.dispatch(action)
     }
 
+    /// Checks the latest Order sync, if the Order already contains products then we mark these as selected
+    ///
+    private func selectExistingProductsInOrderIfNeeded() {
+        let _ = orderSynchronizer.order.items.map { item in
```

Comment:
Perhaps it would be useful to actually pass around the `[OrderItem]` that this map produces, since we're removing the `updatedProductAndVariationIDsInOrder` array anyway?

Reply:
Yes, I made the changes here: https://github.com/woocommerce/woocommerce-ios/pull/9203/commits/ee49133f0600c0b8727acff8d8f0b62b07b49afd where we sync the Order initial state, and also pass around the sync `[OrderItem]` as needed.